### PR TITLE
Remove broken links

### PR DESCRIPTION
**Pull request by freearhey**

### CODE REVIEW

**Review by BellezaEmporium — DISMISSED** [review]

**Review by UltraHDR — DISMISSED** [review]

File: streams/co.m3u [hunk 1]
```diff
@@ -39,8 +39,6 @@ https://play.cdn.enetres.net/B9E570CFE54949C7ABB985D4F8F77F7F021/02267/playlist.
 https://movil.ejeserver.com/live/visiondorada.m3u8
 #EXTINF:-1 tvg-id="CanalVisionDorada.co" status="online",Canal Visión Dorada (720p) [Not 24/7]
 https://video.ejeserver.com/live/visiondorada.m3u8
-#EXTINF:-1 tvg-id="CaracolTV.co" status="blocked",Caracol TV (720p) [Geo-blocked]
-https://mdstrm.com/live-stream-playlist/574463697b9817cf0886fc17.m3u8
```
Comment (BellezaEmporium) [inline streams/co.m3u:43]: this one is GEO

File: streams/us_localbtv.m3u [hunk 2]
```diff
@@ -89,9 +80,6 @@ http://v-ny.theus6tv.duckdns.org/hls/5.3/playlist.m3u8
 #EXTINF:-1 tvg-id="WNYWDT5.us" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",WNYW-DT5 (Decades)
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
 http://v-ny.theus6tv.duckdns.org/hls/5.5/playlist.m3u8
-#EXTINF:-1 tvg-id="WPIXDT1.us" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",WPIX-DT1 (PIX 11)
-#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
-http://v-ny.theus6tv.duckdns.org/hls/11.1/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/us_localbtv.m3u:94]: to be honest... that's the whole file that's down for now.

File: streams/hk.m3u [hunk 3]
```diff
@@ -60,8 +60,6 @@ https://tvp22.sky4k.top/index1.php
 #EXTINF:-1 tvg-id="ViuTV.hk" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36 Edge/12.246",ViuTV (1080p) [Geo-blocked]
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36 Edge/12.246
 https://cdn.hklive.tv/xxxx/99/index.m3u8
-#EXTINF:-1 tvg-id="ViuTV.hk" status="online",ViuTV (720p)
-https://live.astradamy.com/viutv/index.m3u8
```
Comment (Alstruit) [inline streams/hk.m3u:64]: Not broken.

File: streams/nz.m3u [hunk 4]
```diff
@@ -7,8 +7,6 @@ https://juicex.nz/hls/mystream.m3u8
 https://ptvlive.kordia.net.nz/out/v1/3fc2254c865a457c8d7fbbce227a2aae/index.m3u8
 #EXTINF:-1 tvg-id="ParliamentTV.nz" status="online",Parliament TV (1080p)
 https://ptvlive.kordia.net.nz/out/v1/daf20b9a9ec5449dadd734e50ce52b74/index.m3u8
-#EXTINF:-1 tvg-id="Three.nz" status="blocked",Three [Geo-blocked]
-https://livestreamdirect-three.mediaworks.nz/three.m3u8
```
Comment (Alstruit) [inline streams/nz.m3u:11]: Possibly needs header and Geo-blocked.

File: streams/pe.m3u [hunk 5]
```diff
@@ -319,5 +319,3 @@ http://177.53.152.186:8000/play/a00j/index.m3u8
 http://177.53.152.186:8000/play/a00g/index.m3u8
 #EXTINF:-1 tvg-id="RadioyTvFiladelfia.pe" status="online",Radio y TV Filadelfia
 https://streamlive8.hearthis.at/hls/9355343_orig/index.m3u8
-#EXTINF:-1 tvg-id="Latina.pe",Latina (480p)
-https://mdstrm.com/live-stream-playlist/5ce7109c7398b977dc0744cd.m3u8
```
Comment (Alstruit) [inline streams/pe.m3u:323]: Geo-blocked

File: streams/py.m3u [hunk 6]
```diff
@@ -29,8 +29,6 @@ https://d4xunn272arip.cloudfront.net/hls/snt.m3u8
 https://59ce1298bfb98.streamlock.net/somosdeleste/somosdeleste/playlist.m3u8
 #EXTINF:-1 tvg-id="SucesoTV.py" status="online",Suceso TV
 https://59ce1298bfb98.streamlock.net/sucesotv/sucesotv/playlist.m3u8
-#EXTINF:-1 tvg-id="Trece.py" status="online",Trece Paraguay (720p) [Not 24/7]
-https://stream.rpc.com.py/live/trece.m3u8
```
Comment (Alstruit) [inline streams/py.m3u:33]: Works with this link: http://stream.rpc.com.py/live/trece.m3u8

File: streams/ro.m3u [hunk 7]
```diff
@@ -21,8 +21,6 @@ https://atomic.streamnet.ro/atomictv.m3u8
 http://46.4.14.12:9999/btvsvlive/playlist.m3u8
 #EXTINF:-1 tvg-id="CorneaTV.ro" status="timeout",Cornea TV (720p) [Not 24/7]
 http://89.149.30.158:1935/CorneaTV/myStream/playlist.m3u8
-#EXTINF:-1 tvg-id="CredoTV.ro" status="online",Credo TV (720p) [Not 24/7]
-http://cdn.credonet.tv:1935/ctv/smil:livecredo.smil/playlist.m3u8
```
Comment (Alstruit) [inline streams/ro.m3u:25]: Link works.

File: streams/py.m3u [hunk 8]
```diff
@@ -29,8 +29,6 @@ https://d4xunn272arip.cloudfront.net/hls/snt.m3u8
 https://59ce1298bfb98.streamlock.net/somosdeleste/somosdeleste/playlist.m3u8
 #EXTINF:-1 tvg-id="SucesoTV.py" status="online",Suceso TV
 https://59ce1298bfb98.streamlock.net/sucesotv/sucesotv/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/py.m3u:31]:
```suggestion
https://59ce1298bfb98.streamlock.net/sucesotv/sucesotv/playlist.m3u8
#EXTINF:-1 tvg-id="Trece.py" status="online",Trece Paraguay (720p) [Not 24/7]
http://stream.rpc.com.py/live/trece.m3u8
```

File: streams/ro.m3u [hunk 9]
```diff
@@ -21,8 +21,6 @@ https://atomic.streamnet.ro/atomictv.m3u8
 http://46.4.14.12:9999/btvsvlive/playlist.m3u8
 #EXTINF:-1 tvg-id="CorneaTV.ro" status="timeout",Cornea TV (720p) [Not 24/7]
 http://89.149.30.158:1935/CorneaTV/myStream/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/ro.m3u:23]:
```suggestion
http://89.149.30.158:1935/CorneaTV/myStream/playlist.m3u8
#EXTINF:-1 tvg-id="CredoTV.ro" status="online",Credo TV (720p) [Not 24/7]
http://cdn.credonet.tv:1935/ctv/smil:livecredo.smil/playlist.m3u8
```

File: streams/hk.m3u (hunk 3 above)
Comment (freearhey) [reply streams/hk.m3u:64]: Looped video:
image

File: streams/nz.m3u (hunk 4 above)
Comment (freearhey) [reply streams/nz.m3u:11]: The report said that it does not load even in NZ: https://github.com/iptv-org/iptv/issues/10171. I had the same thing with my NZ VPN, it didn't load.

**Review by BellezaEmporium — APPROVED** [review]

**Review by UltraHDR — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Pull request merged**